### PR TITLE
fix: Deserialize filter

### PR DESCRIPTION
**Pull request by kgilpin**

### CODE REVIEW

File: packages/models/src/serialize.js [hunk 1]
```diff
@@ -92,8 +92,16 @@ export function deserializeFilter(filterState) {
...
+  for (const property in filterState) {
+    if (
+      Object.prototype.hasOwnProperty.call(filterState, property) &&
+      filterState[property] === undefined
+    )
+      delete filterState[property];
+  }
+
+  if ('rootObjects' in filterState && filterState.rootObjects !== false) {
+    filter.rootObjects = filterState.rootObjects;
```
Comment (ahtrotta) [inline packages/models/src/serialize.js:104]: Changing it from `filter.declutter.rootObjects` to `filter.rootObjects` is going to break some other code that's looking for `rootObjects` in the `declutter` property. It looks like that's why the build failed. Should we just leave it as `filter.declutter.rootObjects`?

Comment (ahtrotta) [reply packages/models/src/serialize.js:104]: Oh true, I had assumed that the test assertions were reflective of actual code, but that wasn't a good assumption.

**Comment by kgilpin** [comment]
So we should probably update the Components for this as well?

**Review by ahtrotta — APPROVED** [review]
It turns out that the problem was that `serializeFilter` was looking for `rootObjects` in the `declutter` property, so I fixed it.

**Pull request merged**